openshift/ansible/tasks/build_react.yaml

```diff
@@ -13,7 +13,8 @@
     vars:
       GIT_REF: 'pull/{{ PR }}/head'
       build_params: "{{ lookup('template', './templates/build.param.j2') }}"
-
+  - name: cancel any existing builds
+    shell: oc -n "{{ tools_namespace }}" cancel-build bc/{{ react_infra_name }}-{{ version }}
```

Comment:
`{{ react_infra_name }}-{{ version }}` would be digital-gov-frontend-pr-124 or digital-gov-frontend-pr-123. So it will only cancel builds that exist for this pr